Userland/DevTools/UserspaceEmulator/SoftCPU.h

```diff
@@ -1153,7 +1163,10 @@ class SoftCPU final
     void update_code_cache();
 
 private:
+    friend class SoftFPU;
```

Comment:
`class` isn't needed here because you include `"SoftFPU.h"`.

Reply:
Good to know